app/javascript/components/terraform-template-catalog-form/terraform-template-catalog-form.schema.js

```diff
@@ -30,6 +30,70 @@ const basicInformationTabSchema = (availableCatalogs, tenantTree, roleAllows, zo
         id: 'description',
         label: __('Description'),
       },
+      {
+        component: componentTypes.SELECT,
+        id: 'provisioning_entry_point_type',
+        name: 'provisioning_entry_point_type',
+        label: __('Provisioning Entry Point'),
+        initialValue: 'embedded_automate',
+        options: [{ value: 'embedded_automate', label: __('Embedded Automate') }, { value: 'embedded_workflow', label: __('Embedded Workflow') }],
+      },
+      {
+        component: componentTypes.TEXT_FIELD,
+        id: 'provisioning_entry_point_automate',
+        name: 'provisioning_entry_point_automate',
+        label: __('Provisioning Entry Point'),
+        initialValue: '/Service/Generic/StateMachines/GenericLifecycle/provision',
```

Comment:
I don't remember since its been a while that I've looked at this but I believe @agrare said we don't need an API for this because we can use the default entry point data from here https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template_ansible_playbook.rb.